### DISCUSSION TITLE
순수 JS vs React 어떤 것으로 개발해야하는가

### DISCUSSION THREAD

**Discussion by yujo11** [woowacourse/react-lotto#67] [토론해보기]
<img width="1007" alt="스크린샷 2021-05-22 오후 2 08 00" src="https://user-images.githubusercontent.com/61097373/119215266-9aba5500-bb07-11eb-9273-0f418ac31ff4.png">

**Comment by zereight** [comment]
ㅋㅋㅋㅋㅋㅋㅋㅋ
